server/src/main/resources/buildServerResources/buildOverviewZipkinExtension.jsp

```diff
@@ -0,0 +1,13 @@
aCNKLiiP1x7UVM120cQoAADqOrTq1BSwb6zp7ZVtDulDoHNKXiXfrmrnaKBdtwOvn2MgIANAezAtsVLxYLGg6BfmlWlpfVwx1UYPMM120sQoAANK500X74rbNgwgFK0RtBOkthfkhreqiVXZvMccAACpiotND18ydstqcXQohaiNI7yvGxYJkLUteY84BAMzxArpRsbmukXvUa0iHELVPkD5WTHVRU1VrYf8S8xAASogtOLhK0V3XxJ7tNaQLhc55/NLx5qvzZg/uYS3Jbfn3qY4GqwBQfMyM7lLFWY3yAkKIOi5Itg7+UgmSuQT2dFGD1U2ZqwBQMGyPpTkbjGm0FxBC1DhBsjL1eou4waoJ0jbMXQDIOban0ryAxjfLCwghao4otW2wao/sdmUuA0DOMDO6aS4yoyv8auHORf1hbRqsbq3jyY4Gq1Cdd0kBBIDtmbQFWeYF9E5ZfnTnov/AeDmjNVjdOK6QaLAKAKFhZnSTFNN1zfqgbD++c1l+qAb3CR16SZDs/ZF1/D5UsRDzHwA8YmZ04xXn6xr1YVmT0LlsP1iD/ScdjpQgne6iR3Z9FItyPgBAhpgZ3RiFdcEufZPnzmX94TRYBQAP2Cb80YrLfJvRIURhCZItiRwqQbLyeKBikGJJpgYANJBnFXbTe01IPkDBXIdJwX8EiQarANBoHlMc6KJWPFcgQghRrYL01Qar/0dWAKBO7lfs7aJmpNeHaEYXEp1JQaIgfazD1NmDe8zQ8QeKIS5qtgoAkMTvXdQH7nZSgRA1UpDsheIsCdIFLvJ7t8d2NFgFgLbcphit68UfSQVC1ExBsme7P5cgXeYi//fhjgarAGXHvICsEekDpAIhylKQ7FnvdRIka7K6m8L2I/UgM7mHpbRQK2ZGd7VinK4Hj5EOhMinINlkbNtg1bo17EJmcstHpABSmOsFNEHn/zOkAyEKTZTucv9tsGrvkPYlKwCFYa4X0KQimdEhRMUVJGuwup8EaZNYkGiwCpBfrPXO+bEAvUo6EKK8CdLjLmqwau+PrON3b3INkBus+aht22gpshkdQlQeQXrB/bfBqlVIRzoarAKEinkBmRnd1DKY0SFE5RMkK+v7SZDO1PEERV9Hg1WAUDAvIDOjm6ZzFYNEhKjwgmRl/hAJkrUQOjYOGqwC+MG8gFoUM3Ru/ot0IERlEyQr+0+XIE12kf2EtQ9alswAZIJ5AZkb6vlxKy9AiEotSPZMepwE6Wwdj44FqRuZAWgK5gU0UfEzvIAQIphfkGiwCtA8zAtoguJSbBgQIkgXJBqsAjQO20Zh72OvxoYBIYL6BYkGq9nzISkoDA8qxihujFtxQeCw6z8HSJBsnGiw2lzxXyDD8bSbikfJesO520VWDLeSCioiaPxF8qsNVq1C2pnMAMzhDheZ0d1FKhAiyEaU7GSb22DV2gftQ1agpNyiGBX3eASECDwIkp18+8aPeU52NFiFcmBPB26IKyAebxblekYKci9Ijyl6uWh13cUu8kwBKBo2r69UdNd8PxARQoggTEF6XnGk/rim4jwXtbAHyDvmBXSRYn3N7+9hSIcQQT4E6VVF31iQpjiWJUM++TS+oVpb87kPhnTFhndExRUka7B6wuzBPcbqOEgx0NFgFcLHLNvNjG48XkAIERRHkKzB6mlxg1WrlGiwCiFi3a+nu8gNFS8ghAgKKkjms2INVs/R8ceKoYqVyQx4xualNf2dihcQQgTlESR79NG2waoJ0ppkBjLGqh6r0qfhBQQIUXkFqW2D1UNdtBeprA1W32NGZIaZ0Y1XnIcXECBEMFeQrMHqJRKkS3U8QDHM0WAVGo+Z0Vkj0gvxAgKECJIEydrkXytBuk7H3V3UYHVrMgMdxMzoznJ4AQFCBHUIkrVQ+bUFDVahA5gZ3ZmKazSn6PYBCBG0W5TucjRYhfp4THGG4peY0QFCBI0UJBqsQhr3K0YqbsOMDhAiaKYg2d1uLwnSCBfZmH9fsSCZKTW/V5ypufE7UgHtvraQAmiHINFgFW5TbKt5sAMiBAgR+BQkGqyWjxsV39S47664l3QAQgShCNLrihP0x9UVox0bRIuGvfO5SrGxxrmn4iFSAo2Ed0TQSEGyti3DZw/uMUnHfgoTJxqs5hdbdm0bncdpbJ8jHYAQQZ4EyZpXjo0brP5IcaJipYC/8qeM2jyYGd2Figkay5dJByBEkGdBsndGbRusmiCtEeBXpedZhC06scUnkzV2fyUdgBBBkQSpbYPV3oqTXHkbrIaI3TBMU5yNGR0gRFB0QbJeYxdLkH6u44Eu6tZAg1V/2KIS8wI6FzM6QIigbIJkrV+ukSBdq+MeLmqwuhWZyYy3XbTcfgZmdIAQQdkFyZYF32IRN1g9TbETmWka5gVkKxrPw4wOECKA+UXpLvffBqvmibQ3WWkYtvBgguKnmNEBQgSQLkjWYHUfCdJmLlrUQIPV9mNeQOaGejFmdBD0eU8KIFBBelTRS3/c0C6kLtpcCbVhXkCHK9ZXDs9HhAAhAuiYID0XN1hdy0V7XNh8msyc7uiK7srZz3FEBYQIoLGC9Je4waptiG1xNFhti3kB9VRsrhxdhSEd5A3eEUHeBMk2XB4/e3CPMToepzhGsWRJ0/EHxVjl5FZmBiBEANkLUiMbrOZtKfNvYgG6i5kAhTifSQHkXJDeVYzVH1dTDFa0p0VNXt6l3KTood+7GyIECBFAeIL0oWKqizyRrEJ6pSA/zTb9WgcKe/+zT7y8HaBQ8GgOiiZIRWmwasvVr1SM0W96hpEFhAggf4LUtsHqQYpTXD4arJoXkH1newf0IiMJCBFA/gXJljJfLUG6Rsc9XdTPLsQGq+YF9DMXuaG+ysgBQgRQPEGydy03W0iUdowFaccAvprthzrfRW6oeAEBQgRQElG6U4c74warwxXdPHwN8wKapZiEFxAAQMmRIK2a8ed1VSxF5gEAAAAC4P8FGABDGMmeCJOEkgAAAABJRU5ErkJggg=="/>
```

Comment:
😂 gross

Reply:
an SVG with an embedded image? _I think they're doing it wrong 🤦 _